packages/framework-integration-tests/integration/helper/cli-helper.ts

```diff
@@ -17,5 +17,5 @@ export async function nuke(projectPath: string, environmentName = 'production'):
 }
 
 export function start(environmentName = 'local', path: string): ChildProcess {
```

Comment:
```suggestion
export function start(path: string, environmentName = 'local'): ChildProcess {
```
Can we reverse these params so we can call `start(path)` and get the advantage of having default param?